.gitignore

```diff
@@ -4,3 +4,4 @@
 /.idea/*
 !/.idea/codeStyles
 /.kotlin
+local.properties
```

Comment:
This isn't an android project so this isn't needed

Reply:
Fair, I use Android Studio for a few KMP projects and it gets generated by default, which is annoying. I don't think excluding this file has a downside, but I'll revert.